test/unit/components/chart/daily.test.js

```diff
@@ -0,0 +1,105 @@
+/* global chai */
+
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var _ = require('lodash');
+var expect = chai.expect;
+
+var Daily = require('../../../../app/components/chart/daily');
+
+require('tideline/css/tideline.less');
+require('../../../../app/core/less/fonts.less');
+require('../../../../app/style.less');
+window.localStorage.debug = true;
+
+describe('Daily', function () {
+  describe('render', function() {
```

Comment:
Hmm, I imagine these weren't working because of troubles providing all the expected props? I'll play around with it a bit, but in general this might be just be exposure of the fact that the interface between blip and tideline needs work (and since I'm hoping we'll be working on a new tideline in the not-so-distant future, it's probably not worth putting effort into fixing now).